profiles/kentik_snmp/cisco/cisco-catalyst.yml

```diff
@@ -412,6 +412,11 @@ metrics:
         name: cieIfInputQueueDrops
       - OID: 1.3.6.1.4.1.9.9.276.1.1.1.1.11
         name: cieIfOutputQueueDrops
+  - MIB: SNMP-FRAMEWORK-MIB
+    symbols:
+      - OID: 1.3.6.1.6.3.10.2.1.3
+        name: snmpEngineTime
+        tag: Uptime
```

Comment:
@al3kh - this needs to be below the `metric_tags` section. that isn't a global `metric_tags` setting, that's specific to the `cieIfPacketStatsTable` and putting your metric here is going to break the YAML